### PR TITLE
Update dependencies to prodcon build 20180208-05-1367307

### PR DESCRIPTION
**Pull request by natemcmaster**
- Update packages to 20180208-05-1367307 prodcon output
 - Updates corefxlab packages to 0.1.0-preview1-180208-1

Add script to take prodcon manifest and apply it to our dependencies.props. cc @ryanbrandenburg 

Inputs for this change: https://raw.githubusercontent.com/dotnet/versions/738604f6623377659e95ddddca9f1a4a99ea02f2/build-info/dotnet/product/cli/release/2.1/build.xml

### CODE REVIEW

**Comment by natemcmaster** [comment]
No merge because I need to investigate this:
```
C:\dev\aspnet\Universe21\modules\KestrelHttpServer\src\Kestrel.Core\Kestrel.Core.csproj : error NU1605: Detected package downgrade: System.Runtime.CompilerServices.Unsafe from 4.5.0-preview1-26202-03 to 4.5.0-preview1-26126-05. Reference the package directly from the project to select a different version.
C:\dev\aspnet\Universe21\modules\KestrelHttpServer\src\Kestrel.Core\Kestrel.Core.csproj : error NU1605:  Microsoft.AspNetCore.Server.Kestrel.Core -> Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions -> Microsoft.AspNetCore.Protocols.Abstractions -> System.IO.Pipelines 0.1.0-preview1-180207-2 -> System.Buffers.Primitives 0.1.0-preview1-180207-2 -> System.Runtime.CompilerServices.Unsafe (>= 4.5.0-preview1-26202-03)

C:\dev\aspnet\Universe21\modules\KestrelHttpServer\src\Kestrel.Core\Kestrel.Core.csproj : error NU1605:  Microsoft.AspNetCore.Server.Kestrel.Core -> System.Runtime.CompilerServices.Unsafe (>= 4.5.0-preview1-26126-05)
```

**Comment by natemcmaster** [comment]
Package downgrade warning was due to unclean machine state.

:up: :date: with a few more upgrades and fixes.

File: scripts/common.psm1 [hunk 1]
```diff
@@ -51,3 +64,43 @@ function Get-Submodules {
...
+    $settings = New-Object System.XML.XmlWriterSettings
+    $settings.OmitXmlDeclaration = $true
+    $settings.Encoding = New-Object System.Text.UTF8Encoding( $true )
+    $writer = [System.XML.XMLTextWriter]::Create($path, $settings)
+    $xml.Save($writer)
+    $writer.Close()
+}
+
+function LoadXml([string]$path) {
+    Write-Verbose "Reading to $path"
```
Comment (pakrym) [inline scripts/common.psm1]: from?

**Comment by natemcmaster** [comment]
cc @Eilon  / @muratg  - preview1.

**Comment by natemcmaster** [comment]
We will have to keep doing this manually for preview1 because corefxlab packages are manually updated to new prodcon builds. I wanted to write scripts to help make this work better.

**Pull request merged**